master/setup.py

```diff
@@ -453,7 +453,7 @@ def define_plugin_entries(groups):
     'txaio ' + txaio_ver,
     'autobahn ' + autobahn_ver,
     'PyJWT',
-    'distro'
+    'distro;platform_system==="Linux"'
```

Comment:
I wasn't aware of this construct. Is this supported by all pip/setuptools versions?
Does that work with wheels?